lib/dialects/postgres/data-types.js

```diff
@@ -647,6 +647,51 @@ module.exports = BaseTypes => {
     array_oids: []
   };
 
+  function CIDR() {
+    if (!(this instanceof CIDR)) return new CIDR();
+    BaseTypes.CIDR.apply(this, arguments);
+  }
+  inherits(CIDR, BaseTypes.CIDR);
+
+  CIDR.parse = function parse(value) {
```

Comment:
No need to add parse if we are not modifying this value, I think just defining oids should be enough for all three of these types

Reply:
@sushantdhiman Done.